pages/home/install/docker.mdx

```diff
@@ -29,4 +29,4 @@ where config.yaml is the path to the [configuration file](/home/configure/templa
 
 ## Worker
 
-Arcade does not currently provide a prebuilt Worker image. However, you can build your own following the [FastAPI Guide](/home/serve-tools/build-a-worker-with-fastapi) and running your app in a Docker container.
+Arcade now provides a base Worker image that you can use to build your custom Worker image. Follow the [Build custom worker images with docker](/home/serve-tools/docker-worker) guide to create your own Worker image using Arcade's base Worker image.
```

Comment:
the fast api worker example no longer exists, so this now references the docker worker docs